utils/misc.mli

```diff
@@ -374,7 +374,6 @@ module Color : sig
     | BG of color (* background *)
     | Bold
     | Reset
-
```

Comment:
The blank line deleted here seems like an accident. It is not hugely important, but it makes it harder to track the provenance of code using `git blame`, so if you can reinstate the blank line that would be great. :-)